openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -268,12 +270,18 @@ def write_project_to_op(project: dict, dbcon: AvalonMongoDB) -> UpdateOne:
         {
             "code": project_code,
             "fps": float(project["fps"]),
-            "resolutionWidth": int(project["resolution"].split("x")[0]),
-            "resolutionHeight": int(project["resolution"].split("x")[1]),
             "zou_id": project["id"],
         }
     )
 
+    match_res = re.match(r"(\d+)x(\d+)", project["resolution"])
+    if match_res:
+        project_data['resolutionWidth'] = match_res.group(1)
+        project_data['resolutionHeight'] = match_res.group(2)
```

Comment:
```suggestion
        project_data['resolutionWidth'] = int(match_res.group(1))
        project_data['resolutionHeight'] = int(match_res.group(2))
```
Previously these were integers.